source/operations/server-side-encryption/configure-minio-kes-hashicorp.rst

```diff
@@ -16,279 +16,24 @@ Server-Side Object Encryption with Hashicorp Vault Root KMS
 .. |KES-git| replace:: :minio-git:`Key Encryption Service (KES) <kes>`
 .. |KES|     replace:: :abbr:`KES (Key Encryption Service)`
 
-MinIO Server-Side Encryption (SSE) protects objects as part of write operations,
-allowing clients to take advantage of server processing power to secure objects
-at the storage layer (encryption-at-rest). SSE also provides key functionality
-to regulatory and compliance requirements around secure locking and erasure.
+MinIO Server-Side Encryption (SSE) protects objects as part of write operations, allowing clients to take advantage of server processing power to secure objects at the storage layer (encryption-at-rest). 
+SSE also provides key functionality to regulatory and compliance requirements around secure locking and erasure.
 
-MinIO SSE uses |KES-git| and an
-external root Key Management Service (KMS) for performing secured cryptographic
-operations at scale. The root KMS provides stateful and secured storage of 
-External Keys (EK) while |KES| is stateless and derives additional cryptographic
-keys from the root-managed |EK|. 
+MinIO SSE uses |KES-git| and an external root Key Management Service (KMS) for performing secured cryptographic operations at scale. 
+The root KMS provides stateful and secured storage of External Keys (EK) while |KES| is stateless and derives additional cryptographic keys from the root-managed |EK|. 
 
-This procedure does the following:
+.. cond:: container
 
-- Configure |KES| to use  
-  `Hashicorp Vault <https://www.vaultproject.io/>`__ as the root |KMS|.
+   .. include:: /includes/container/steps-configure-minio-kes-hashicorp.rst
 
-- Configure MinIO to use the |KES| instance for supporting |SSE|.
-  
-- Configure automatic bucket-default 
-  :ref:`SSE-KMS <minio-encryption-sse-kms>` and 
-  :ref:`SSE-S3 <minio-encryption-sse-s3>`.
+.. cond:: linux
 
-Prerequisites
--------------
+   .. include:: /includes/linux/steps-configure-minio-kes-hashicorp.rst
 
-.. _minio-sse-vault-prereq-vault:
+.. cond:: macos
```

Comment:
Are you doing a Windows slice?

Reply:
![image](https://user-images.githubusercontent.com/14060842/182644985-233e3da5-b04c-4ac3-922e-ec6c3d60b794.png)